test/catch2/live_restore/api/test_live_restore_fs_open_file.cpp

```diff
@@ -106,36 +106,38 @@ TEST_CASE("Live Restore fs_open_file", "[live_restore],[live_restore_open_file]"
 
     SECTION("fs_open - Directory")
     {
+        /*
+         * WiredTiger never creates directories. The user must do this outside of WiredTiger. This
+         * means all cases where the directory doesn't exist in the destination we return ENOENT
+         * These cases include when:
+         */
         WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh = nullptr;
 
-        // If the folder doesn't exist return ENOENT.
+        // The directory doesn't exist in the source or destination
```

Comment:
Although it is a bullet point, and not a complete sentence on its own, it still feels like it should end with a fullstop. And maybe have a bullet or `*` at the beginning to make it clear it is a list item?

(And whatever you may do, do the same for all of them below)